docs/maintainer-guide/issues.md

```diff
@@ -0,0 +1,82 @@
+    * Label requests for changes to existing features (new rule options, new configuration options, etc.) with the "enhancement" and "proposal" labels
+    * Label requests for completely new features (new rules, supporting a new file format, etc.) with the "feature" and "proposal" labels
+    * Use an appropriate label for the part of the project the issue refers to:
+      * "build" - related to commands run during a build (testing, linting, release scripts, etc.)
+      * "cli" - related to command line input or output, or to `CLIEngine`
+      * "core" - related to internal APIs
+      * "documentation" - related to content on eslint.org
+      * "infrastructure" - related to resources needed for builds or deployment (VMs, CI tools, bots, etc.)
+1. Once it's clear what type of issue it is, make sure all of the relevant information is provided:
+  * **Bugs**: See [bug reporting guidelines](/docs/developer-guide/contributing/reporting-bugs)
+  * **New Rules:** See [rule proposal guidelines](/docs/developer-guide/contributing/new-rules)
+  * **Other Changes:** See [change proposal guidelines](http://eslint.org/docs/developer-guide/contributing/changes)
+1. Next steps:
+  * **Questions:** answer the question and close the issue when the conversation is over.
+  * **Bugs:** if you can verify the bug, add the "accepted" label and ask if they would like to submit a pull request.
+  * **New Rules:** if you are willing to champion the rule (meaning you believe it should be included in ESLint core), then add a comment saying you will champion the rule and add an "accepted" label. As the champion, it's your job to guide the rule creation process until it's complete, so only champion a rule that you have time to implement or help another contributor implement it.
+  * **Duplicates:** if you can verify the issue is a duplicate, add a comment mentioning the duplicate issue (such as, "Duplicate of #1234") and close the issue.
+1. Regardless of the above, always leave a comment. Don't just add labels, engage with the person who opened the issue by asking a question (request more information if necessary) or stating your opinion of the issue. If it's a verified bug, ask if the user would like to submit a pull request.
+
+**Note:** Don't add an "accepted" label to an issue unless it's a bug that you've been able to reproduce and verify (you're sure it's a bug). The "accepted" label will be added by a project admin if it's appropriate for the roadmap.
+
+## Evaluating an Issue
+
+If you're a project admin, then you'll need to evaluate incoming requests for inclusion in the formal roadmap. You should label an issue as "accepted" when all of the following are true:
+
+1. The feature or enhancement is in scope for the project and should be added to the roadmap
+1. You are committed to including the change within the next year
+1. You are reasonably certain of who will do the work
```

Comment:
What about `accepted`+`help needed`? As in you know that somebody from outside of the team will need to do the change/feature. We currently do this all the time.